3-extra/1-radio-stations.js

```diff
@@ -26,6 +53,17 @@
  */
 // `getStations` goes here
 
+function getStations() {
```

Comment:
This looks great.
As an extra exercise - can you re-write this using the `filter` array method?